mautic/mautic
# [UX] Make the "Show opened by default" label concise, removing the first word

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The label “Show opened by default” has been updated to “Open by Default” for enhanced brevity and clarity. This streamlined label more effectively communicates the setting’s purpose, allowing users to quickly understand that the related feature will be active upon initial display. This refinement contributes to a more intuitive user interface.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Reports
3. Create new
4. Look at the label

![image](https://github.com/mautic/mautic/assets/116097999/d11ae645-e5fb-4887-8ca1-fa2226e17fcc)
